BackofficeBundle/Validator/Constraints/BlockNodePatternValidator.php

```diff
@@ -67,4 +71,22 @@ public function validate($node, Constraint $constraint)
             }
         }
     }
+    protected function getRefBlock(AreaContainerInterface $container)
+    {
+        $areas = $container->getAreas();
+        if (count($areas) > 0){
+            foreach ($areas as $area) {
+                $this->getRefBlock($area);
+            }
+        } else {
+            $blocks = $container->getBlocks();
+            if (count($blocks) > 0){
+                foreach ($blocks as $block) {
+                    if($block['nodeId'] !== NodeInterface::TRANSVERSE_NODE_ID) {
```

Comment:
you should make the other test : if the nodeId == 0, then the block is saved inside the current node